substrate-node/runtime/src/lib.rs

```diff
@@ -832,7 +832,16 @@ impl_runtime_apis! {
         }
 
         fn authorities() -> Vec<AuraId> {
-            Aura::authorities().into_inner()
+            let authorities: Vec<AuraId> = ValidatorSet::validators()
+                .iter()
+                .map(|account| {
+                    let mut bytes = [0u8; 32];
+                    bytes.copy_from_slice(&account.encode());
+                    sr25519::Public::from_raw(bytes).into()
+                })
+                .collect();
```

Comment:
With the log removed, also remove the let binding, and the `;` after `collect()` so you just return the result of the expression.